[2024-05-11T23:11:13.884+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gtaV_reviews_project.extract manual__2024-05-11T23:11:12.840869+00:00 [queued]>
[2024-05-11T23:11:13.891+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gtaV_reviews_project.extract manual__2024-05-11T23:11:12.840869+00:00 [queued]>
[2024-05-11T23:11:13.892+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-05-11T23:11:13.906+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): extract> on 2024-05-11 23:11:12.840869+00:00
[2024-05-11T23:11:13.911+0000] {standard_task_runner.py:60} INFO - Started process 292 to run task
[2024-05-11T23:11:13.913+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gtaV_reviews_project', 'extract', 'manual__2024-05-11T23:11:12.840869+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/gtaV_reviews_pipeline.py', '--cfg-path', '/tmp/tmp1mlucnjq']
[2024-05-11T23:11:13.915+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask extract
[2024-05-11T23:11:13.958+0000] {task_command.py:423} INFO - Running <TaskInstance: gtaV_reviews_project.extract manual__2024-05-11T23:11:12.840869+00:00 [running]> on host 93cdc42157bb
[2024-05-11T23:11:14.024+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gtaV_reviews_project' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2024-05-11T23:11:12.840869+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-11T23:11:12.840869+00:00'
[2024-05-11T23:11:16.852+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 241, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/gtaV_reviews_pipeline.py", line 25, in extract
    data = pd.read_excel('/opt/airflow/data/testFile.xlsx')
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/excel/_base.py", line 478, in read_excel
    io = ExcelFile(io, storage_options=storage_options, engine=engine)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/excel/_base.py", line 1496, in __init__
    ext = inspect_excel_format(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/excel/_base.py", line 1371, in inspect_excel_format
    with get_handle(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/io/common.py", line 868, in get_handle
    handle = open(handle, ioargs.mode)
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/data/testFile.xlsx'
[2024-05-11T23:11:16.865+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=gtaV_reviews_project, task_id=extract, execution_date=20240511T231112, start_date=20240511T231113, end_date=20240511T231116
[2024-05-11T23:11:16.881+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 10 for task extract ([Errno 2] No such file or directory: '/opt/airflow/data/testFile.xlsx'; 292)
[2024-05-11T23:11:16.901+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-05-11T23:11:16.924+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
